mne/io/pick.py

```diff
@@ -15,51 +15,94 @@
                      _check_option)
 
 
-def get_channel_type_constants():
-    """Return all known channel types.
+def get_channel_type_constants(include_defaults=False):
```

Comment:
@larsoner could you have a look at how I've altered this function?  As I understand it: sometimes we want the returned dict to have minimal entries (values that should *always* be present in `info['chs']` for a given channel type, regardless of the acquisition system used).  Other times, we want additional values because we use the returned dict to pre-populate an info dict for synthetic data (e.g., in `create_info`).  Does that understanding accurately reflect all of the relevant use cases for this kind of dictionary?  If so, does the approach I've taken here seem optimal?